src/lib/mina_block/precomputed_block.ml

```diff
@@ -125,22 +127,21 @@ let of_block ~logger
   ; delta_transition_chain_proof = Header.delta_block_chain_proof header
   ; accounts_accessed
   ; accounts_created
+  ; body_reference = Header.body_reference header
   }
 
 (* NOTE: This serialization is used externally and MUST NOT change.
     If the underlying types change, you should write a conversion, or add
     optional fields and handle them appropriately.
 *)
+(* But if you really need to update it, see output of CLI command:
+   `dune runtest src/lib/transition_frontier/tests/ 2> block.txt` *)
```

Comment:
Rethinking this somewhat: can you tweak this so that it only prints when you ask it to (ie. make it an executable rather than an inline test that does the printing)? It's a constant battle trying to keep the unit test output to what's relevant, I don't think we want this every time we run the tests.